client/src/data_layer_configs/lead_service_lines_by_parcel_config.ts

```diff
@@ -0,0 +1,108 @@
+import {DataSourceType, FeaturePropertyDataType, LegendInfo, MapLayer, PopupInfo, TileDataLayer} from '@/model/data_layer';
+import {FillLayer} from 'mapbox-gl';
+import {colorMapToBuckets, tileServerHost} from '@/util/data_layer_util';
+
+const DEFAULT_NULL_COLOR = '#d3d3d3';
+
+/**
+ * Maps legend buckets to the hex values.
+ */
+const LEGEND_COLOR_MAPPING = [
+  0,
+  '#9fcd7c',
+  5,
+  '#f7e5af',
+  38,
+  '#f4a163',
+  99.7,
+  '#ff5934',
+  99.9,
+  '#d73819',
+];
+
+/**
+ * Maps feature property values to colors to be rendered on the map.
+ */
+const INTERPOLATION_COLOR_MAPPING = [
```

Comment:
If you wait for https://github.com/BlueConduit/open-data-platform/pull/64, you can use the helper function and not have to have two separate functions here

Reply:
Ack!